@xen-orchestra/backups/writers/_MixinBackupWriter.js

```diff
@@ -33,6 +34,7 @@ exports.MixinBackupWriter = (BaseClass = Object) =>
         })
       } catch (error) {
         warn(error)
+        Task.warning(`error while cleaning the backup folder : ${error.toString()}`)
```

Comment:
Does this PR includes @ByScripts's changes?

Reply:
Yes